### ISSUE TITLE
[Bug]: Not launching on steam deck

### ISSUE THREAD

**Issue by Mory45** [Lyall/P5StrikersFix#3]
When installing all the files the game doesnt launch, tried modifying the .ini file to see if changing some settings would fix it but nothing worked, everything set to false, removing the games from the game folder launches the game, to install the files i just did "browse local files" from steam and pasted the files there

**Comment by GODSWOMB** [comment]
Have you added this line:
WINEDLLOVERRIDES="dinput8=n,b" %command%
to your launch options for P5S?

**Comment by Mory45** [comment]
yes, I followed the instructions on the post, tried it with the command and without it but nothing worked, I copy and paste the command directly from the instructions

**Comment by GODSWOMB** [comment]
Another thing I would try is switching the compatibility to GE-Proton. I installed it quite a while ago so I forgot how to install it for Steam Deck, but it fixes a lot of issues with the game.

After that, you'll want to add this command to the launch options:
GST_PLUGIN_FEATURE_RANK=protonaudioconverterbin:NONE %command%

I'm not sure if using both of these %command% lines will break something, I have yet to test anything.

**Comment by Mory45** [comment]
so doing some testing with the commands, if I add the command WINEDLLOVERRIDES="dinput8=n,b" %command% as the first command the game doesnt launch, if I add it as the second or last command the game does launch but the only thing that the mod changes is the analog movement, I didnt notice any other changes like the intro skip or the resolution change (just tested to see if anything changed) FOV and UI looked the same, I also have the command gamemoderun mangohud %command% without it the game launches but not to the main menu

**Comment by GODSWOMB** [comment]
Unfortunately I don't think there'll be an easy resolution for this. I'm running into the same issues. There might be some compatibility issues, as P5Sfix doesn't get loaded at all. (The analog fix doesn't actually work. If you switch your gamepad sensitivity down and try to move, you can only move in 4 directions)

I tried installing the latest version of GE-Proton (9.16) but I can't get past the startup loading screen.

Probably have to wait until official Proton support (which will probably be never atp), or hopefully Lyall will have some kind of solution.

**Comment by Mory45** [comment]
for me I use GE-Proton 8-25 since none of the GE-Proton 9 versions worked for me

**Comment by GODSWOMB** [comment]
That was the version I was using before installing 9.16. Try and see if you can find some documentation relating to GE-Proton <9.00 that could prove useful. For now though, playing without this fix is your best bet, or playing on a Windows device.

**Comment by termdisc** [comment]
> Another thing I would try is switching the compatibility to GE-Proton. I installed it quite a while ago so I forgot how to install it for Steam Deck, but it fixes a lot of issues with the game.
> 
> After that, you'll want to add this command to the launch options: GST_PLUGIN_FEATURE_RANK=protonaudioconverterbin:NONE %command%
> 
> I'm not sure if using both of these %command% lines will break something, I have yet to test anything.

The `protonaudioconverterbin` setting is already covered by GE-Proton8-12 and newer.

> I tried installing the latest version of GE-Proton (9.16) but I can't get past the startup loading screen.

Without the fix mod, GE 9-12 works on Deck (running Bazzite) with full voiced cutscenes, while 9-13 works on my Bazzite desktop. GE 9-14 through 9-16 do not work on either Deck or desktop with each of these grinding to a halt before the Atlus logo shows up.

To the point of the original issue, GE 9-12 and 9-13 do not work with this fix for me. I have attached my Proton log for this one with 9-12, which does not even boot into the game at all. I'll dig into it when I get some more free time.

[steam-1382330.log](https://github.com/user-attachments/files/17405404/steam-1382330.log)

**Comment by termdisc** [comment]
I tried using v1.0.0 on my Bazzite desktop, and it works. Attached are various logs from Proton + P5StrikersFix

[P5StrikersFix.log](https://github.com/user-attachments/files/17423265/P5StrikersFix.log)

[steam-1382330.log](https://github.com/user-attachments/files/17423268/steam-1382330.log)

For contrast, `P5StrikersFix.log` when attempting to run v1.0.1 has no data at all.

**Comment by termdisc** [comment]
Found a solution that worked for me: install `vcrun2022` via Protontricks (and skip over any SHA256 mismatches).

I came to this by noticing the backtrace in my logs directed to `msvcp140`, which is a VC++ runtime file. The game already has `vcrun2019` installed by Steam, so I decided to install something more recent. 

I double checked the solution by deleting my prefix, attempting to run the game, installing `vcrun2022`, then run the game one more time.

**Comment by GODSWOMB** [comment]
> Found a solution that worked for me: install `vcrun2022` via Protontricks (and skip over any SHA256 mismatches).
> 
> I came to this by noticing the backtrace in my logs directed to `msvcp140`, which is a VC++ runtime file. The game already has `vcrun2019` installed by Steam, so I decided to install something more recent.
> 
> I double checked the solution by deleting my prefix, attempting to run the game, installing `vcrun2022`, then run the game one more time.

Just tested, and this solution works.
Use GE-8.25 up to 9.04 (?), with only the launch command for P5SFix.
(I tried testing out other GE-Proton versions to check for compatibility, but I think I ended up triggering the Denuvo activation limit. Oops.)
Anyways, feel free to close this thread if this solution works for you, op.

**Comment by Luis-Gdcr** [comment]
I tried the fix installing the vcrun2022 via Protontricks and it all seemed to work fine, but the animated cutscenes have no audio in them, is there any solution to fix that?

**Comment by termdisc** [comment]
> I tried the fix installing the vcrun2022 via Protontricks and it all seemed to work fine, but the animated cutscenes have no audio in them, is there any solution to fix that?

Stock Proton is missing a fix for that.

I have to use GE-Proton9-12 on my Deck and GE-Proton9-13 on my desktop (both with Bazzite) to have fully voiced cut scenes. Anything more recent either has unvoiced audio or just won’t load at all.

**Comment by Carter0108** [comment]
> I tried using v1.0.0 on my Bazzite desktop, and it works. Attached are various logs from Proton + P5StrikersFix
> 
> [P5StrikersFix.log](https://github.com/user-attachments/files/17423265/P5StrikersFix.log)
> 
> [steam-1382330.log](https://github.com/user-attachments/files/17423268/steam-1382330.log)
> 
> For contrast, `P5StrikersFix.log` when attempting to run v1.0.1 has no data at all.

Thought I'd chime in here. Nothing I did would make the game even boot with v1.0.1 installed but it launched first time no issues with v1.0.0. This is using Proton GE 8.25.

**Comment by termdisc** [comment]
@Carter0108 did you follow the instructions in the next comment? https://github.com/Lyall/P5StrikersFix/issues/3#issuecomment-2421191962

**Comment by lyndonguitar** [comment]
> Found a solution that worked for me: install `vcrun2022` via Protontricks (and skip over any SHA256 mismatches).
> 
> I came to this by noticing the backtrace in my logs directed to `msvcp140`, which is a VC++ runtime file. The game already has `vcrun2019` installed by Steam, so I decided to install something more recent.
> 
> I double checked the solution by deleting my prefix, attempting to run the game, installing `vcrun2022`, then run the game one more time.

Thank you so much. this absolutely worked for me

Also, for others using other launch options/fix with Proton and want to combine the launch options, For example 

`GST_PLUGIN_FEATURE_RANK=protonaudioconverterbin:NONE %command%"
`
+
`WINEDLLOVERRIDES="dinput8=n,b" %command%
`
You just need to place them next to each other but only do **ONE** %command% at the end. Order does not matter.

So it will be 
`GST_PLUGIN_FEATURE_RANK=protonaudioconverterbin:NONE WINEDLLOVERRIDES="dinput8=n,b" %command%
`
This worked for me.

I'm also using lossless scaling so i had to put `~/lsfg` BEFORE %command%, if you put it at the beginning, game will not launch
At the end for (ProtonAudioFix + P5StrikersFix + LSFG), it should be like 

`GST_PLUGIN_FEATURE_RANK=protonaudioconverterbin:NONE WINEDLLOVERRIDES="dinput8=n,b" ~/lsfg %command%
`
Note that i am using GEProton 9-5 for all of this

**Comment by MiamiCereal** [comment]
> > I tried using v1.0.0 on my Bazzite desktop, and it works. Attached are various logs from Proton + P5StrikersFix
> > [P5StrikersFix.log](https://github.com/user-attachments/files/17423265/P5StrikersFix.log)
> > [steam-1382330.log](https://github.com/user-attachments/files/17423268/steam-1382330.log)
> > For contrast, `P5StrikersFix.log` when attempting to run v1.0.1 has no data at all.
> 
> Thought I'd chime in here. Nothing I did would make the game even boot with v1.0.1 installed but it launched first time no issues with v1.0.0. This is using Proton GE 8.25.

Exactly the same. Nothing I do can get this to work

**Comment by NULL-SIX** [comment]
> [@Carter0108](https://github.com/Carter0108) did you follow the instructions in the next comment? [#3 (comment)](https://github.com/Lyall/P5StrikersFix/issues/3#issuecomment-2421191962)

Honestly this just gives an error every time I try to run the game
